### PR TITLE
Upload crds via init container

### PR DESCRIPTION
**Pull request by kaweue**

### CODE REVIEW

**Comment by kaweue** [comment]
recheck ha

**Comment by kaweue** [comment]
recheck ha

File: cmd/crdsloader/BUILD.bazel [hunk 1]
```diff
@@ -0,0 +1,54 @@
+load("@io_bazel_rules_docker//container:container.bzl", "container_image", "container_push")
+
+container_image(
+    name = "crdsloader-crds-layer",
+    # References container_pull from WORKSPACE (above)
```
Comment (katrybacka) [inline cmd/crdsloader/BUILD.bazel:5]: you mean refers?

File: test/env/run_e2e_tests.sh [hunk 2]
```diff
@@ -22,14 +22,11 @@ E2E_TEST_SUITE=${E2E_TEST_SUITE:-aio}
...
+    kubectl apply -f -
 
-kubectl apply -f deploy/cluster_role.yaml
-kubectl apply -f deploy/cluster_role_binding.yaml
-
-operator-sdk test local --verbose ./test/e2e/$E2E_TEST_SUITE --image "$operator_image" --go-test-flags "-timeout=30m -parallel=8"
-
-kubectl delete -f deploy/cluster_role.yaml
-kubectl delete -f deploy/cluster_role_binding.yaml
+go test -v ./test/e2e/$E2E_TEST_SUITE/... -namespacedMan test/env/deploy/empty.yaml -globalMan test/env/deploy/empty.yaml -root $DIR -timeout=30m -parallel=8 -skipCleanupOnError=false
```
Comment (katrybacka) [inline test/env/run_e2e_tests.sh:29]: what is this empty.yaml used for? it looooks confusing :smile_cat:

Comment (kaweue) [reply test/env/run_e2e_tests.sh:32]: fixed

File: cmd/crdsloader/BUILD.bazel (hunk 1 above)
Comment (kaweue) [reply cmd/crdsloader/BUILD.bazel]: removed comment

**Pull request merged**